contributors/design-proposals/node/troubleshoot-running-pods.md

```diff
@@ -235,37 +249,28 @@ The status of a Debug Container is reported in a new field in `v1.PodStatus`:
 ```
 type PodStatus struct {
         ...
-        DebugStatuses []DebugStatus
-}
-
-type DebugStatus struct {
-        Name string
-        Command []string
-        Args []string
-        // Set only for Debug Containers
-        DebugContainerStatus v1.ContainerStatus
+        EphemeralContainerStatuses []v1.ContainerStatus
 }
 ```
 
-Initially this will be populated only for Debug Containers, but there's interest
-in tracking status for traditional exec in a similar manner. Ideally we can
-report both types of user intervention into a container with a single new type.
+This status is only populated for Debug Containers, but there's interest in
+tracking status for traditional exec in a similar manner.
 
-Note that `Command` and `Args` must be tracked in the status object because
-there is no spec for Debug Containers or exec. These must either be made
+Note that `Command` and `Args` would have to be tracked in the status object
+because there is no spec for Debug Containers or exec. These must either be made
 available by the runtime or tracked by the kubelet. For Debug Containers this
 could be stored as runtime labels, but the kubelet currently has no method of
 storing state across restarts for exec. Solving this problem for exec is out of
 scope for Debug Containers, but we will look for a solution as we implement this
 feature.
```

Comment:
@smarterclayton was there something were you mentioning that allowed connecting data to running containers?